src/main/java/org/folio/rest/impl/InventoryHelper.java

```diff
@@ -450,14 +398,13 @@ private Error buildErrorWithParameter(String value, ErrorCodes errorCode) {
       .withParameters(parameters);
   }
 
-  private String buildProductIdQuery(ProductId productId, Map<String, String> productTypes) {
-    return String.format("(identifiers adj \"\\\"identifierTypeId\\\": \\\"%s\\\"\" " +
-        "and identifiers adj \"\\\"value\\\": \\\"%s\\\"\")",
-      productTypes.get(productId.getProductIdType().toString()),
-      productId.getProductId());
+  private String buildProductIdQuery(ProductId productId) {
+    return String.format(
+        "(identifiers adj \\\"\\\\\\\"identifierTypeId\\\\\\\": \\\\\\\"%s\\\\\\\"\\\" \" + and identifiers adj \"\\\"value\\\": \\\"%s\\\"\")",
```

Comment:
extra escaping can be removed for the identifierTypeId and its value

Reply:
master version is
```
return String.format("(identifiers adj \"\\\"identifierTypeId\\\": \\\"%s\\\"\" " +
        "and identifiers adj \"\\\"value\\\": \\\"%s\\\"\")"
```

Reply:
done